### ISSUE TITLE
official quay.io/coreos/flannel:v0.8.0 docker image has root-ca certificate problem

### ISSUE THREAD

**Issue by spacexnice** [coreos/flannel#796]
## Problem
official quay.io/coreos/flannel:v0.8.0 docker image has root-ca certificate problem. which is root ca is not valid. And has been fix in [commit](https://github.com/coreos/flannel/commit/53d1855d7eead6e519f43b75cec725b2121e3557). 

Does it possible to cherry-pick it on v0.8.0? @tomdee

**Comment by roffe** [comment]
correct me if i'm wrong, but isn't the idea you mount the CA's from HOST into flannel so you don't have to build them into the image.

like the flannel-wrapper does in coreOS

```text
        --volume usr-share-certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
        --mount volume=usr-share-certs,target=/usr/share/ca-certificates \
```

**Comment by oilbeater** [comment]
@roffe that means flannel containers rely on host settings, I don't think it's a good practice. 

flannel:v0.6 image does have CAs in image and aws/ali-vpc backend works fine by official kubernetes daemonset yaml, but v0.8 is broken. That's why I commit this change.

**Comment by tomdee** [comment]
This will be fixed in the v0.9.0 release (which has an RC available now)
